app/controllers/admin/users_controller.rb

```diff
@@ -170,6 +171,16 @@ def remove_org_credits
       Credit.remove_from(org, amount)
     end
 
+    def set_related_reactions
+      user_article_ids = @user.articles.ids
+      user_comment_ids = @user.comments.ids
+      @related_vomit_reactions = Reaction.where(reactable_type: "Comment", reactable_id: user_comment_ids,
+                                                category: "vomit")
+        .or(Reaction.where(reactable_type: "Article", reactable_id: user_article_ids, category: "vomit"))
```

Comment:
Sorry, I should've included this on the example code I gave, but I think we'll also want an `.or` for vomit reactions on the user, too.

Reply:
No worries! I almost added that in, but I thought that when you `vomit` on a User, it also vomits on their Articles, so those `vomit`s should fall within the query for Articles too, correct @Zhao-Andy?
<img width="468" alt="Screen Shot 2020-08-28 at 12 58 10 PM" src="https://user-images.githubusercontent.com/32834804/91605711-22cf4d80-e92e-11ea-886e-5a5abf038e4d.png">

Reply:
It actually just lowers the score of their articles, instead of creating vomit reactions for all of their articles. Maybe we should update the language in that message.

Reply:
Okay, good to know! I think we definitely should -- should we make a card for this and add it to the project board or should I address this in this PR?

Reply:
Let's make a card in the project 👍

Reply:
Will do!